ts/src/test/tests.ts

```diff
@@ -966,95 +966,115 @@ class testMainClass extends baseMainTestClass {
+                                this.assertStaticError (exchange.parseToNumeric (sanitizedNewOutput) === exchange.parseToNumeric (sanitizedStoredOutput), messageError, storedOutput, newOutput, assertingKey);
+                                return true;
+                            } else {
+                                this.assertStaticError (convertAscii (newOutputString) === convertAscii (storedOutputString), messageError, storedOutput, newOutput, assertingKey);
+                                return true;
+                            }
                         } else {
                             this.assertStaticError (convertAscii (newOutputString) === convertAscii (storedOutputString), messageError, storedOutput, newOutput, assertingKey);
                             return true;
                         }
                     } else {
-                        this.assertStaticError (convertAscii (newOutputString) === convertAscii (storedOutputString), messageError, storedOutput, newOutput, assertingKey);
-                        return true;
-                    }
-                } else {
-                    if (this.lang === "C#") { // tmp fix, stil failing with the "1.0" != "1" error
-                        const stringifiedNewOutput = exchange.numberToString (sanitizedNewOutput);
-                        const stringifiedStoredOutput = exchange.numberToString (sanitizedStoredOutput);
-                        this.assertStaticError (stringifiedNewOutput.toString () === stringifiedStoredOutput.toString (), messageError, storedOutput, newOutput, assertingKey);
-                    } else {
-                        const numericNewOutput =  exchange.parseToNumeric (newOutputString);
-                        const numericStoredOutput = exchange.parseToNumeric (storedOutputString);
-                        this.assertStaticError (numericNewOutput === numericStoredOutput, messageError, storedOutput, newOutput, assertingKey);
+                        if (this.lang === "C#") { // tmp fix, stil failing with the "1.0" != "1" error
+                            const stringifiedNewOutput = exchange.numberToString (sanitizedNewOutput);
+                            const stringifiedStoredOutput = exchange.numberToString (sanitizedStoredOutput);
+                            this.assertStaticError (stringifiedNewOutput.toString () === stringifiedStoredOutput.toString (), messageError, storedOutput, newOutput, assertingKey);
+                        } else {
+                            const numericNewOutput =  exchange.parseToNumeric (newOutputString);
+                            const numericStoredOutput = exchange.parseToNumeric (storedOutputString);
+                            this.assertStaticError (numericNewOutput === numericStoredOutput, messageError, storedOutput, newOutput, assertingKey);
+                        }
                     }
                 }
             }
+        } catch (e) {
+            if (this.info) {
```

Comment:
the exception might be completely unrelated with the assertion, so I think that printing this line will only add to the confusion no?

Reply:
i think i don't follow what you said. exception 'might be' in some theoretical case, but in most cases it's exactly related to assertion, and moreover, it's under `this.info`, so there is no room of confusion when printing the exact assertion reason, for whom who use `--info`